modules/FindMagnum.cmake

```diff
@@ -534,6 +534,10 @@ foreach(_component ${Magnum_FIND_COMPONENTS})
                 find_package(SDL2)
                 set_property(TARGET Magnum::${_component} APPEND PROPERTY
                     INTERFACE_LINK_LIBRARIES SDL2::SDL2)
+                if(CORRADE_TARGET_EMSCRIPTEN)
+                    # TODO: give me INTERFACE_LINK_OPTIONS or something, please
+                    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -s USE_SDL=2")
```

Comment:
I need to verify that this doesn't bloat Emscripten apps that are not using SDL2. Apart from that, I have no other comment. Thanks for the contribution!